backend/core-api/src/modules/auth/utils.ts

```
@@ -1,6 +1,103 @@
+  console.log({ fromEmail });
+
+  sendgridMail.setApiKey(SENDGRID_API_KEY);
+  return await sendgridMail
+    .send({
+      from: fromEmail,
+      to: toEmail,
+      subject: 'Login to erxes',
+      html,
+    })
+    .then(
+      () => {},
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)